DB_SETUP_POSTGRESQL.md

```diff
@@ -0,0 +1,88 @@
+# Install PostgreSQL
+
+Download [PostgreSQL](https://www.postgresql.org/download/)
+
+## Linux (Ubuntu/Debian)
+
+```bash
+sudo sh -c 'echo "deb https://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+sudo apt-get update
+sudo apt-get -y install postgresql 
+```
+
+# Initial Set up
+Once installed you need to run a least minimum initialization of your database server:
+
+- Create a database
+- Create a user (representing the Signum Node)
+- Give user access to the database
+
+Run `psql` in your command line; You should see something like this:
+
+```
+user@computer:~$ psql
+psql (16.1 (Ubuntu 16.1-1.pgdg20.04+1), server 12.17 (Ubuntu 12.17-1.pgdg20.04+1))
+Type "help" for help.
+
+user=#
+```
+> This is on Linux, but it might be somehow similar on Windows, or Mac.
+
+Then execute the following commands in the `psql` prompt.
+
+## For Main Net
+
+The main net should be the preferred setup, unless you want to do some development.
+
+```sql
```

Comment:
Flyway migrations for Postgresql since version 15 will generate error: 

> permission denied for schema public

The following commands are sufficient to create a database and a user for that database:
```sql
-- Create the user (choose another password if you want)
CREATE USER signumnode WITH PASSWORD 's1gn00m_n0d3';

-- Create the database
CREATE DATABASE signum OWNER=signumnode;
```